skbio/io/format/tests/test_stockholm.py

```diff
@@ -0,0 +1,148 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from six import assertRaisesRegex
+
+import unittest
+
+from skbio import TabularMSA
+from skbio.io import StockholmFormatError
+from skbio.io.format.stockholm import (_stockholm_to_tabular_msa,
+                                       _stockholm_sniffer)
+from skbio import Protein, DNA, RNA
+from skbio.util import get_data_path
+
+
+class TestStockholmSniffer(unittest.TestCase):
+    def setUp(self):
+        self.positives = [get_data_path(e) for e in [
+            'stockholm_extensive',
+            ]]
+
+        self.negatives = [get_data_path(e) for e in [
+            'stockholm_data_only',
```

Comment:
Can you rename this file to something like `stockholm_no_header` or `stockholm_missing_header` to better reflect this test file's purpose? `stockholm_data_only` could be interpreted as meaning "a stockholm file without metadata" (which is valid).